src/DistortableCollection.js

```diff
@@ -288,6 +290,81 @@ L.DistortableCollection = L.FeatureGroup.extend({
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
     return json;
   },
+
+  // generateExportJson(allImages = false) {
```

Comment:
Oh no, wait - was this supposed to be commented out?

Reply:
I can revert if this was an error, sorry I should have looked one more time before merging!

Reply:
Ah, i see, it was one of three alternatives. The correct one is still in place. Can you remove these comments in a follow-up PR? Thank you!!

Reply:
Yes, I will remove the comments. Many thanks!